package main.go

```diff
@@ -1,7 +1,9 @@
 package main
 
-import "fmt"
+import (
+	"github.com/kyokomi/emoji"
+)
```

Comment:
this could be one line, as one package imported. smth. like: **import "github.com/kyokomi/emoji"**